src/todos/todos.service.ts

```
@@ -58,52 +58,87 @@ export class TodosService {
       console.log(error)
-      throw new UnauthorizedException("Invalid TODO Creation")
+
+      return {
+        result: 'error',
+        message: 'INVALID_REQUEST'
+      }
     }
   }
 
-  async findAll(user_id: number) {
+  async findAll(user_id: number): Promise<SuccessResponse | FailResponse | ErrorResponse>  {
```

ESLint

[error] Replace `user_id:·number):·Promise<SuccessResponse·|·FailResponse·|·ErrorResponse>··` with `⏎····user_id:·number,⏎··):·Promise<SuccessResponse·|·FailResponse·|·ErrorResponse>·`

(prettier/prettier)